app/views/admin/users/index/controls/_export.html.erb

```diff
@@ -1,5 +1,20 @@
-<a href="<%= export_admin_users_path %>.csv">
-  <svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
-    <path d="M13 0L18 5V19.008C17.9997 19.2712 17.895 19.5235 17.7088 19.7095C17.5226 19.8955 17.2702 20 17.007 20H0.993C0.730378 19.9982 0.479017 19.8931 0.293218 19.7075C0.107418 19.5219 0.00209465 19.2706 0 19.008V0.992C0 0.444 0.445 0 0.993 0H13ZM10 10V6H8V10H5L9 14L13 10H10Z" fill="#404040" />
-  </svg>
-</a>
+<button type="button" class="js-export-csv-modal-trigger c-btn w-100 align-left" data-modal-trigger data-modal-title="Download member data" aria-label="Download member data" data-modal-size="small" data-modal-content-selector="#export-csv-modal">
+  <%= crayons_icon_tag(:export, class: "c-btn__icon") %>
+</button>
+
+<div id="export-csv-modal" class="hidden">
+  <div class="flex flex-col gap-4">
+    <p>
+      Your data will be downloaded as a Comma Separated Values (.csv) file.
+    </p>
+    <p>
+      Values listed are Name, Username, Email address, Status, Joining date, Last activity, and Organizations.
+    </p>
+    <div class="flex gap-2">
+      <%= link_to "Download", "#{export_admin_users_path}.csv", class: "c-cta c-cta--branded self-end" %>
+      <button class="js-export-csv-modal-cancel crayons-btn crayons-btn--ghost">
+        Dismiss
```

Comment:
```suggestion
        Cancel
```
In the design we have "Cancel" (although tbh I wonder if we need the button at all since we also have the X at the top 🤷‍♀️ )